erpnext/stock/doctype/delivery_note/delivery_note.py

```
@@ -1001,7 +1002,7 @@ def make_delivery_trip(source_name, target_doc=None, kwargs=None):
 
 
 @frappe.whitelist()
-def make_installation_note(source_name, target_doc=None, kwargs=None):
+def make_installation_note(source_name: str, target_doc: Document | None = None, kwargs: dict | None = None):
```

Ruff (0.15.1)

[warning] Unused function argument: `kwargs`

(ARG001)